src/component/header/HeaderActionButtons.tsx

```diff
@@ -0,0 +1,41 @@
+import { MouseEvent } from 'react';
+
+import Button from '../elements/Button';
+
+interface HeaderActionButtonsProps {
+  onApply: (event?: MouseEvent) => void;
+  onCancel: (event?: MouseEvent) => void;
+  applyLabel?: string;
+  cancelLabel?: string;
+  applyDataTestId?: string;
+}
+
+export default function HeaderActionButtons(props: HeaderActionButtonsProps) {
+  const {
+    onApply,
+    onCancel,
+    applyLabel = 'Apply',
+    cancelLabel = 'Cancel',
+    applyDataTestId,
+  } = props;
+  return (
+    <div style={{ marginLeft: 10, marginRight: 10, display: 'flex' }}>
+      <Button.Done
+        onClick={onApply}
+        borderRadius="0.3rem"
+        data-test-id={applyDataTestId}
```

Comment:
We normally don't need a test id for these cases. You should be able to use `text=label` in the test's selector

Reply:
@targos 
thank you, https://github.com/cheminfo/nmrium/pull/1467/commits/4f1df76c14a2ab9640059d834ad47a06ded867f7